plugins/prco/src/dataSources.ts

```diff
@@ -0,0 +1,508 @@
+import type { ManagedCollectionFieldInput } from "framer-plugin"
+import * as v from "valibot"
+import {
+    AlexaSchema,
+    ClippingImageAsImageSchema,
+    ClippingsSchema,
+    FeaturedImagesForClSchema,
+    ImageSizesSchema,
+    MediaInfoSchema,
+    MediaKitsSchema,
+    type PrCoItem,
+    PressReleasesSchema,
+    SocialSchema,
+    TagsSchema,
+} from "./api-types"
+
+const API_URL = "https://api.pr.co/v1"
+
+export interface PrCoDataSource {
```

Comment:
Nit, not sure how others feel, but I'd use `Prco` instead of `PrCo` (in every name). `PrCo` trips me up when I read the types, and it makes the `Field` or `DataSource` part stand out less.

```suggestion
export interface PrcoDataSource {
```